### PR TITLE
Update EKS Add-ons docs to better align with AWS Docs site

### PR DESCRIPTION
**Pull request by bryan-aguilar**
This PR aims to better align the ADOT Dev portal with https://docs.aws.amazon.com/eks/latest/userguide/opentelemetry.html

The installation and update/remove docs have been updated to match instructions from the linked aws docs. These instructions include more thorough console instructions. Added the sample app page that was not present in the ADOT dev portal. Add troubleshooting page that is not present on ADOT dev portal.

### CODE REVIEW

File: src/docs/getting-started/adot-eks-add-on/installation.mdx [hunk 1]
```diff
@@ -5,45 +5,61 @@ description:
...
++ You have met the [ADOT prerequisites](/docs/getting-started/adot-eks-add-on/requirements)\.
 
-This section will go into detail for the second and third steps. Alternatively, the second and third steps can be simplified, and combined into one, using the EKS add-ons advanced configuration feature. For more information on advanced configuration for ADOT, see [Add-on Advanced Configuration](/docs/getting-started/adot-eks-add-on/add-on-configuration). For more information on ADOT Collector deployment using advanced configuration, see [Add-on Advanced Configuration: Collector Deployment](/docs/getting-started/adot-eks-add-on/add-on-configuration-collector-deployment).
+------
+## AWS Management Console 
 
-## Leverage EKS Add-Ons to deploy the ADOT Operator to your Amazon EKS cluster
+Install the ADOT Amazon EKS add\-on to your Amazon EKS cluster using the following steps:
```
Comment (humivo) [inline src/docs/getting-started/adot-eks-add-on/installation.mdx:15]:
```suggestion
Install the ADOT Amazon EKS add-on to your Amazon EKS cluster using the following steps:
```

File: src/docs/getting-started/adot-eks-add-on/sample-app.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: 'Deploy a sample application to test the AWS Distro for OpenTelemetry Collector'
+description:
+    This page provides an a sample app that can be used with the Advanced Configuration
+path: '/docs/getting-started/adot-eks-add-on/target-allocator'
```
Comment (humivo) [inline src/docs/getting-started/adot-eks-add-on/sample-app.mdx:5]:
```suggestion
path: '/docs/getting-started/adot-eks-add-on/sample-app'
```

File: src/docs/getting-started/adot-eks-add-on/sample-app.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+To apply the traffic generator and sample application, do the following steps\. Make sure that you have satisfied the prerequisites and completed the 
+procedure in [Install the AWS Distro for OpenTelemetry \(ADOT\) Operator](/docs/getting-started/adot-eks-add-on/installation)\. 
+
+1. Download the `traffic-generator.yaml` file to your computer\. You can also [view the file](https://github.com/aws-observability/aws-otel-community/blob/master/sample-configs/traffic-generator.yaml) on GitHub\.
+
+   ```
+   curl -O https://raw.githubusercontent.com/aws-observability/aws-otel-community/master/sample-configs/traffic-generator.yaml
+   ```
+
+1. In `traffic-generator.yaml`, make sure that the second `kind` value reflects your mode\. For more information,
```
Comment (humivo) [inline src/docs/getting-started/adot-eks-add-on/sample-app.mdx:23]: I believe this should be `2.` and so on

Comment (bryan-aguilar) [reply src/docs/getting-started/adot-eks-add-on/sample-app.mdx]: good catch, fixed.

**Review by humivo — APPROVED** [review]

**Pull request merged**